frontend/src/renderer/components/Process/Dropbox.svelte

```diff
@@ -11,17 +11,6 @@
     export let singleItem = true;
     export let showError = false;
 
-    const truncate = function(text) {
-        const start = text.slice(0, 16 - 7);
-        const end = text.slice(-7);
-
-        if (start.length + end.length < text.length) {
-            return start + '…' + end;
-        } else {
-            return text;
-        }
-    };
-
```

Comment:
Removed this erroneously added function. Was planning on truncating image names if they're long, but it's not that simple.